### PR TITLE
1657389 Add descriptions for SEO

### PR DESCRIPTION
**Pull request by TimShererWithAquent**
## Add descriptions for SEO

@BillWagner - Bill, I wanted to call these out to you before we did very many. Because this is programming reference, I went with a little more direct tone. There are fewer "call to action" descriptions. 

**If you think the tone is too formal and should be more chatty, please let me know.**

For anyone that GitHub notifies, this is for [1657389](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1657389).

You're welcome but not required, according to KhairunJ's guidance, to review these proposed meta descriptions. I'll leave this PR open for 48 hours and address any feedback. After that, we'll sign off for merge. Thanks!

Please keep in mind our 100-160 char length for descriptions.

Fixes #1657389

### CODE REVIEW

File: docs/csharp/programming-guide/classes-and-structs/constants.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Constants - C# Programming Guide"
+description: Constants in C# are immutable values, known at compile time, which do not change for the life of the program. Only C# built-in types can be constants.
```
Comment (BillWagner) [inline docs/csharp/programming-guide/classes-and-structs/constants.md:3]: Constants are a compile time construct, so I'm suggesting a change here:

```suggestion
description: Constants in C# are compile-time literal values, which do not change once the program is compiled. Only C# built-in types can be constants.
```

**Review by BillWagner — CHANGES_REQUESTED** [review]
Thanks @TimShererWithAquent I had one suggestion, and then this is ready to merge.

**Comment by TimShererWithAquent** [comment]
@BillWagner - thanks for the fix

This area has another 400 descriptions. You're welcome to look them over, but that sounds like a lot of work for you. 

If you do and need extra time to do it, we can match your schedule. 

Please merge when you get a chance.

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
Thanks @TimShererWithAquent 

I'll keep my 👀 open for new PRs. The scope of one PR per 20 articles is a nice pace. Thanks for asking.

**Pull request merged**